src/server/health.mjs

```diff
@@ -0,0 +1,70 @@
+import * as Fs from "fs";
+import fetch from "node-fetch";
+const errorHandler = (res, result) => {
+  res.writeHead(500, { "Content-Type": "application/json" });
+  res.write(JSON.stringify(result));
+  res.end();
+};
+
+let checkHealth = async (res) => {
+  let output = {
+    env_config: false,
+    app_file: false,
+    wasm_file: false,
+    api_check: false,
+  };
+  try {
+    let indexFile = "dist/hyperswitch/index.html";
+    let envFile = "dist/hyperswitch/env-config.js";
+
+    let data = Fs.readFileSync(indexFile, { encoding: "utf8" });
+    if (data.includes(`<script src="/env-config.js"></script>`)) {
+      output.env_config = true;
+    }
+    if (data.includes(`<div id="app"></div>`)) {
+      output.app_file = true;
+    }
+    if (
+      data.includes(`<script type="module" src="/wasm/euclid.js"></script>`)
+    ) {
+      output.wasm_file = true;
+    }
+
+    let envString = Fs.readFileSync(envFile, { encoding: "utf8" });
+    const match = envString.match(/apiBaseUrl:\s*"([^"]*)"/);
+
+    // Check if match is found and extract the value
+    const apiBaseUrl = match ? match[1] : null;
+
+    let api = await fetch("https://sandbox.hyperswitch.io/health");
```

Comment:
Do we need to check sandbox explicitly here? Shouldn't we check for `apiBaseUrl/health`